src/schemata/definitions/field.yml

```
@@ -8,14 +8,10 @@ allOf:
     properties:
       ui:widget:
         type: string
-        enum: ['checkbox', 'switch', 'textarea']
+        enum: ['checkbox', 'switch', 'textarea', 'input']
         description: |
-          A flag indicating the widget variant for fields of type `string` or `boolean`.
-
-          Boolean fields can be displayed as either a checkbox or a switch to toggle between a binary state. The default appearance is a checkbox.
-          Fields of type `string` can be configured as a textarea element, which allows for multi-line text. The default appearance is a single-line input.
-
-          If a field type does not support a certain `ui:widget`, the setting is ignored.
+          A flag indicating the widget variant for fields of type `string` or `boolean`. All other field types ignore this setting.
+          Boolean fields can be displayed as either a checkbox or a switch to toggle between a binary state. The default appearance is a checkbox. Fields of type `string` can be configured as a multi-line textarea element. Their default appearance of a single-line input.
```

```suggestion
          A flag indicating the widget variant for fields of type `string` or `boolean`. All other field types ignore this setting.
          Boolean fields can be displayed as either a checkbox or a switch to toggle between a binary state. The default appearance is a checkbox. Fields of type `string` can be configured as a textarea element. Their default appearance is a text field.
```

textarea is a multi-line text field while a single-line input is a text field or a text input. What do you think?

---

I personally prefer the current change, because it references the actual enum values and is explicit about their difference by mentioning single/multi-line. Wdyt?

I did fix a typo though

```diff
- Their default appearance of a single-line input.
+ Their default appearance is a single-line input.
```

---

Right.. I just felt it was redundant. but since it provides clarity to the docs reader, then LGTM.